src/algebra/lie/quotient.lean

```diff
@@ -127,6 +127,20 @@ instance lie_quotient_lie_algebra : lie_algebra R (quotient I) :=
                             rw ←submodule.quotient.mk_smul, },
                    apply congr_arg, apply lie_smul, } }
 
+/-- `lie_submodule.quotient.mk` as a `lie_module_hom`. -/
+@[simps]
+def mk' : M →ₗ⁅R,L⁆ quotient N :=
+{ to_fun := mk, map_lie' := λ r m, rfl, ..N.to_submodule.mkq}
```

Comment:
@ocfnash, we don't already have this in another file, right?

Reply:
Correct, we do not.

(This Lie quotient stuff is very old and would benefit from some attention.)